### PR TITLE
feat: Was made Basic System Design

### PR DESCRIPTION
**Pull request by v-liudwinski**
First pushing of Basic System Design. All details in README.md

### CODE REVIEW

File: EntitiesCollection/EntitiesCollection.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿using Entities.Entities;
+
+namespace EntitiesCollection;
+
+public class EntitiesCollection
```
Comment (Vitalii-Shmidt) [inline EntitiesCollection/EntitiesCollection.cs]: Does it collection or rather wrapper?

File: EntitiesCollection/EntitiesCollection.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+{
+    private readonly List<IEntity> _entities;
+
+    public EntitiesCollection()
+    {
+        _entities = new List<IEntity>();
+    }
+
+    public void AddEntity(IEntity entity) => _entities.Add(entity);
```
Comment (Vitalii-Shmidt) [inline EntitiesCollection/EntitiesCollection.cs]: what happens if you will add a null to your collection?

File: Entity/Entities/Bird.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+public class Bird : IEntity, ICreature, IFlyable
+{
+    public Bird(int id, string name, double maxFlyingHeight)
+    {
+        Id = id;
+        Name = name;
+        MaxFlyingHeight = maxFlyingHeight;
+    }
+
+    public int Id { get; set; }
```
Comment (Vitalii-Shmidt) [inline Entity/Entities/Bird.cs]: you have many common functionality from one class to another, so are you sure that you gonna you interfaces or rather base class?

File: Entity/Entities/Bird.cs [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    public string Name { get; set; }
+    public double MaxFlyingHeight { get; set; }
+    
+    public override string ToString()
```
Comment (Vitalii-Shmidt) [inline Entity/Entities/Bird.cs]: same. common logic of ToString() method from one class to another

File: EntitiesCollection/EntitiesCollection.cs (hunk 1 above)
Comment (v-liudwinski) [reply EntitiesCollection/EntitiesCollection.cs]: I guess wrapper

File: EntitiesCollection/EntitiesCollection.cs (hunk 2 above)
Comment (v-liudwinski) [reply EntitiesCollection/EntitiesCollection.cs]: fixed

File: Entity/Entities/Bird.cs (hunk 4 above)
Comment (v-liudwinski) [reply Entity/Entities/Bird.cs]: fixed

File: Entity/Entities/Bird.cs (hunk 3 above)
Comment (v-liudwinski) [reply Entity/Entities/Bird.cs]: fixed

**Comment by Vitalii-Shmidt** [comment]
LGTM!

**Pull request merged**